Container Security/eks-implementation-guide.md

```diff
@@ -22,14 +22,21 @@ The Falcon Container sensor for Linux extends runtime security to container work
 
 ## Pre-requisites
 
+- Existing AWS Account and VPC
```

Comment:
Is a VPC truly needed?  Does this mean LUMOS must be added into an existing VPC? Does one need to be created, and if so, with what permissions?

Reply:
You're probably right here as eksctl is going to create the VPC for the cluster. Could probably remove this requirement.